build.sbt

```diff
@@ -122,25 +122,24 @@ lazy val sdkJavaTestKit = project
       "java.lang"))
   .settings(Dependencies.sdkJavaTestKit)
 
-lazy val sdkSpring = project
-  .in(file("sdk/spring-sdk"))
+lazy val sdkJavaSpring = project
+  .in(file("sdk/java-sdk-spring"))
```

Comment:
can we adjust the val name with the folder name?

Reply:
Yep

Reply:
@aludwiko, I changed the sbt modules names to follow the package name. I think it looks better now, but I realise that the main reason for it was to have them all grouped when running `sbt projects`. 

We can revert it if we find it inconvenient. 

btw, I also change the tckJava/tckScala to javaTck/scalaTck

Reply:
grouped in the sense to start typing `sdk...` and wait for auto-completion?

Reply:
yep, but I think this is more of habit. We can learn the other way around.